src/v8-orinoco/v8-orinoco.svg

```diff
@@ -0,0 +1 @@
+<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="57.397%" y1="67.448%" x2="9.782%" y2="-7.749%" id="a"><stop stop-opacity=".234" offset="0%"/><stop offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path d="M100 20c48.896 0 65 37.766 65 80s-16.104 80-65 80-65-32.543-65-80 16.104-80 65-80z" fill="#464646"/><path d="M152.287 119a5 5 0 0 0 4.33-7.5L140.32 83.273l-22.516 13L130.925 119h21.362zM57.402 135.782a5 5 0 0 0 4.33 7.5h32.594v-26H68.083l-10.681 18.5zM90.311 45.218a5 5 0 0 0-8.66 0L65.354 73.445l22.517 13 13.12-22.727-10.68-18.5z" fill="#E8E8E8"/><circle fill="#518AEC" cx="100" cy="100" r="33"/><path d="M132.492 105.797L106 79.305l13.042-6.26C127.487 79.022 133 88.867 133 100c0 1.978-.174 3.915-.508 5.797zM70.84 84.535l44.625 44.625A32.855 32.855 0 0 1 100 133c-18.225 0-33-14.775-33-33a32.855 32.855 0 0 1 3.84-15.465z" fill="url(#a)" opacity=".104"/><path d="M115 120h37.287a6 6 0 0 0 3.317-1l-10.856 18.953A16 16 0 0 1 130.864 146H115v5l-14-18 14-18v5zM75.18 102.99l-18.644 32.292a6 6 0 0 0-.792 3.372l-10.986-18.878a16 16 0 0 1-.027-16.047l7.932-13.739-4.33-2.5 22.588-3.124 8.589 21.124-4.33-2.5zM109.82 77.01L91.178 44.718a6 6 0 0 0-2.525-2.372l21.842-.075a16 16 0 0 1 13.911 8l7.932 13.739 4.33-2.5-8.588 21.124-22.589-3.124 4.33-2.5z" fill="#FFF"/></g></svg>
```

Comment:
@mathiasbynens Is it supposed to be an ellipse and not a circle?

Reply:
yeah